commands/audit/auditparams.go

```diff
@@ -7,14 +7,17 @@ import (
 
 type AuditParams struct {
 	xrayGraphScanParams *services.XrayGraphScanParams
-	workingDirs         []string
-	installFunc         func(tech string) error
-	fixableOnly         bool
-	minSeverityFilter   string
+	// common params to all scan routines
```

Comment:
```suggestion
	// Common params to all SCA scan routines
```
comments should start with capital